src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -422,6 +423,9 @@ public void deleteInstance(Infrastructure infrastructure, String instanceId) {
                                               .orElseThrow(() -> new RuntimeException("ERROR unable to find instance with ID: '" +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "ERROR unable to find instance with ID: '" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1192)